crates/staged-sync/src/utils/init.rs

```diff
@@ -68,3 +68,50 @@ pub fn init_genesis<DB: Database>(db: Arc<DB>, chain: ChainSpec) -> Result<H256,
     tx.commit()?;
     Ok(hash)
 }
+
+#[cfg(test)]
+mod tests {
+
+    use super::init_genesis;
+    use crate::utils::init::WriteMap;
+    use reth_db::mdbx::test_utils::create_test_rw_db;
+    use reth_primitives::{
+        Header, GOERLI, GOERLI_GENESIS, MAINNET, MAINNET_GENESIS, SEPOLIA, SEPOLIA_GENESIS,
+    };
+
+    #[test]
+    fn success_init_genesis_mainnet() {
+        let db = create_test_rw_db::<WriteMap>();
+
+        let mut chain_spec = MAINNET.clone();
+        chain_spec.genesis_hash = Header::default().hash_slow();
```

Comment:
the genesis hash in the `chain_spec` is not being used in `init_genesis` so I don't think setting it in these tests is necessary